src/ahbicht/expressions/expression_resolver.py

```diff
@@ -122,3 +135,64 @@ async def _package_async(self, package_key_token: Token) -> Tree[Token]:
         # the package_expression is not None because that's the definition of "has been resolved successfully"
         tree_result = parse_condition_expression_to_tree(resolved_package.package_expression)  # type:ignore[arg-type]
         return tree_result
+
+
+# pylint: disable=no-self-use, invalid-name
+class TimeConditionTransformer(Transformer):
+    """
+    The TimeConditionEvaluator replaces "time conditions" (aka "UB1", "UB2", "UB3") with evaluatable format constraints.
+    This is not what BDEW suggests us to do in the "Allgemeine Festlegungen". BDEW says that "UBx" conditions have to be
+    expanded just like packages: For example "UB1" shall be expanded to "([931] ∧ [932] [490]) ⊻ ([931] ∧ [933] [491])".
+
+    This just doesn't work out for multiple reasons:
+
+    1. Other than _all_ the other requirement constraints RC 490, 491, 492 and 493 are self-referencing.
+    While "normal" RCs act like "You have to provide this ("Foo") if the other thing ("Bar") meets a requirement",
+    the 49x  RCs are of kind "This ("Foo") has to meet certain requirements (f.e. the end of a German day), regardless
+    of the other things ("Bar"). So the usual requirement constraint evaluation approach ("Search for Bar and derive
+    from there what it means for this (Foo)") won't work and is overly complicated, too.
+    Also, if the pseudo requirement constraint 490-493 evaluates to false, this does _not_ mean that the data must not
+    be provided which is also a different behaviour compared to usual requirement constraints.
+
+    2. No one who understands the concept of datetime+offset and is able to parse datetime+offset nowadays cares, if you
+    use ("x datetime" with an "0 offset") or ("x+z datetime" that has a "z offset") instead. It's just BDEWs home-brewed
+    EDIFACT serialization rule that, for no real reason, restricts us to set the offset z to "+00:00" always.
+    https://imgflip.com/i/65giq5
+
+    AHBicht won't restrict you to use datetime offset=="+00:00". Because we don't care and you should not care.
+    Also this default transformer won't expand the UBx conditions into something which is overly complicated and in the
```

Comment:
```suggestion
    AHBicht won't restrict you to use datetime offset=="+00:00". Because we don't care and you should not care.
    The scope of AHBicht is to evaluate expressions and data, not to obey pointless BDEW EDIFACT rules.
    Also this default transformer won't expand the UBx conditions into something which is overly complicated and in the
```